packages/editor-ui/src/components/ActivationModal.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<Modal
+		:name="WORKFLOW_ACTIVE_MODAL_KEY"
+		width="460px"
+		minWidth="350px"
+		title="Workflow activated"
+	>
+		<template v-slot:content>
+			<p :class="$style.p">{{triggerContent}}</p>
+			<p :class="[$style.p, $style.spaced]"><span :class="$style.emphasised">These executions will not show up immediately in the editor</span>, but you can see them in the <a @click="showExecutionsList">execution list</a>.</p>
+		</template>
+
+
+		<template v-slot:footer="{ close }">
+			<div :class="$style.footer">
+				<el-checkbox :value="checked" @change="handleCheckboxChange">Don't show again</el-checkbox>
+				<n8n-button @click="close" label="Got it"/>
+			</div>
+		</template>
+	</Modal>
+</template>
+
+<script lang="ts">
+
+import Modal from '@/components/Modal.vue';
+import { WORKFLOW_ACTIVE_MODAL_KEY, EXECUTIONS_MODAL_KEY, LOCAL_STORAGE_ACTIVATION_FLAG } from '../constants';
+import { INodeUi } from '../Interface';
+import mixins from 'vue-typed-mixins';
+import { workflowHelpers } from '@/components/mixins/workflowHelpers';
+
+
+export default mixins(workflowHelpers).extend({
```

Comment:
definitely don't need the entire weight of `workflowHelpers` to simply get the node name. would rather have kept the regex replace. recommend a stateless helper function instead.